javascript/javascript2/week3/warmUpExcecise/main.js

```diff
@@ -0,0 +1,114 @@
+// 1. Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+console.log("I am ready to learn javascript. Where is my coffee!!")
+setTimeout(function () {
+    console.log('After 2.5 seconds -- OMG!! What is this? Where is my coffee!!');
+}, 2500);
+
+
+/* 2. Create a function that takes 2 parameters: delay and stringToLog. Calling this function should log out the stringToLog after delay seconds. Call the function you have created with some different arguments.
+*/
+
+function parametersTo(delay, stringToLog) {
+    setTimeout(() => {
+        console.log(stringToLog);
+    }, delay * 1000)
+}
+parametersTo(0, 'This is logged after 0 seconds');
+parametersTo(3, 'This is logged after 3 seconds');
+parametersTo(5, 'This is logged after 5 seconds');
+
+
+/* 3. Create a button in html. When clicking this button, use the function you created in the previous task to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+*/
+
+const btn = document.querySelector("#button_1").addEventListener("click", function onClick() {
```

Comment:
Since you're not doing anything with this variable anymore, there is no need to create it :) 

```suggestion
document.querySelector("#button_1").addEventListener("click", function onClick() {
```